dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaExtension.kt

```diff
@@ -176,4 +176,42 @@ constructor(
      */
     fun ProcessIsolation(configure: ProcessIsolation.() -> Unit = {}): ProcessIsolation =
         objects.newInstance<ProcessIsolation>().apply(configure)
+
+
+    //region deprecated properties
+    /**
+     * ```
+     * dokka {
+     *   // DEPRECATED
+     *   suppressInheritedMembers.set(true)
+     *
+     *   // Replace with:
+     *   dokkaPublications.configureEach {
+     *     suppressInheritedMembers.set(true)
+     *   }
+     * }
+     * ```
+     * @see DokkaPublication.suppressInheritedMembers
+     */
+    @Deprecated("Moved to DokkaPublication#suppressInheritedMembers")
```

Comment:
Should these `Deprecation`s include `ReplaceWith("dokkaPublications.configureEach { suppressInheritedMembers.set(true) }")`?

Reply:
If you can think of a way of making that work, then that would be lovely. But I don't think it's possible ([internal discussion](https://jetbrains.slack.com/archives/C05NTFCF9NV/p1727283728288829)).

```kotlin
// I want IJ to quick-replace it with this:
publications.configureEach { enabled.set(true) }

// But it gets replaced with this :(
publications.configureEach { enabled }.set(true) // ERROR Unresolved reference.
```

Reply:
TIL `ReplaceWith` preserves the call expression